src/clib/pioc_sc.c

```diff
@@ -239,13 +253,15 @@ int CalcStartandCount(int basetype, int ndims, const int *gdims, int num_io_proc
     int tiorank;
     int ioprocs;
     int tioprocs;
-    int mystart[ndims], mykount[ndims];
+    int mystart[ndims], mycount[ndims];
     long int pknt;
-    long int tpsize=0;
+    long int tpsize = 0;
 
+    /* ??? */
```

Comment:
We are trying to find start and count indices for each iotask such that each task has approximately blocksize data to write (read).   The number of iotasks participating in the operation is blocksize/global_size.